client/src/pages/UserProfile.jsx

```diff
@@ -45,32 +57,142 @@ export function UserProfile(props) {
     getReviews();
   };
 
+  const handleTabChange = (e, newValue) => {
+    setCurrentTab(newValue);
+  };
+
+  const userProfileChange = () => {
```

Comment:
Does this also need to update the database?

Reply:
Yea I intentionally didn't do that right now and just did UI. I'll add ticket for it to get back to this. I want to relook at how we are sharing user state between different react components and if we change it, it can affect the way this function is written.